tip-wlan/resources/environments/dev-amazon-tip.yaml

```diff
@@ -116,10 +116,10 @@ wlan-spc-service:
     sslKeystorePassword: mypassword
     sslTruststorePassword: mypassword
 nginx-ingress-controller:
-  enabled: true
+  enabled: false
```

Comment:
Why is nginx disabled? We may have issues with UI

Reply:
that came from cicd deployment, we had conflicts with nginx-ingress already installed there for monitoring stuff

Reply:
Hmmm... In that case:
- either we need to resolve the conflict, so both can run
- or we need to configure the other nginx to forward the traffic coming for a service on our cluster appropriately.

Reply:
I think they're so many problems with these recipes and it's important to keep changes small and manageable, not trying to fix all issues at once.
I'll revert this particular file to the original state as its not related to the minikube setup.